Resilio Connect API/JSMonitor/app.js

```diff
@@ -36,3 +36,23 @@ setTimeout(function() {console.log(enumerateAgents());}, 5000);
 
 console.log(findArrayDiff([1, 5, 17, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160], [1, 5, 17, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 165]));
 
+var prevList;
+var updatedList = [];
```

Comment:
What would happen the first time periodicAgentUpdate() is called?  Will it detect new Agents? Are they actually new?